src/SonarQube.Client/Api/SonarQubeService.cs

```diff
@@ -231,20 +231,15 @@ public async Task<SonarQubeQualityProfile> GetQualityProfileAsync(string project
                 },
                 token);
 
-        public async Task<IList<SonarQubeIssue>> GetSuppressedIssuesAsync(string projectKey, CancellationToken token)
-        {
-            const string statusResolved = "RESOLVED";
-            var result = await InvokeRequestAsync<IGetIssuesRequest, SonarQubeIssue[]>(
+        public async Task<IList<SonarQubeIssue>> GetSuppressedIssuesAsync(string projectKey, CancellationToken token) =>
+            await InvokeRequestAsync<IGetIssuesRequest, SonarQubeIssue[]>(
                 request =>
                 {
                     request.ProjectKey = projectKey;
-                    request.Statuses = statusResolved;
+                    request.Statuses = "RESOLVED"; // Resolved issues will be hidden in SLVS
```

Comment:
We should create an enumeration for this, removed the constant for consistency as we have other literals in this class